sdk/monitor/microsoft-opentelemetry-exporter-azuremonitor/tests/test_base_exporter.py

```diff
@@ -40,23 +41,28 @@ def setUpClass(cls):
             "APPINSIGHTS_INSTRUMENTATIONKEY"
         ] = "1234abcd-5678-4efa-8abc-1234567890ab"
         cls._base = BaseExporter()
-        cls._base.storage.path=STORAGE_PATH
+        cls._base.storage.path = STORAGE_PATH
+        cls._envelopes_to_export = [TelemetryItem(name="Test", time=datetime.now())]
 
     @classmethod
     def tearDownClass(cls):
         shutil.rmtree(TEST_FOLDER, True)
 
-    def setUp(self):
-        if os.path.exists(STORAGE_PATH):
-            for filename in os.listdir(STORAGE_PATH):
-                file_path = os.path.join(STORAGE_PATH, filename)
-                try:
-                    if os.path.isfile(file_path) or os.path.islink(file_path):
-                        os.unlink(file_path)
-                    elif os.path.isdir(file_path):
-                        shutil.rmtree(file_path, True)
-                except OSError as e:
-                    print("Failed to delete %s. Reason: %s" % (file_path, e))
+    def shutdown(self):
```

Comment:
Is `shutdown` a special function related to `pytest`?

Reply:
Added by mistake I was testing something else here, reverting to use setUp